app/views/home/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<header class="site-header" role="banner">
+  <div class="wrapper">
+      <p class="site-title" >Haxxor News</p>
+      <nav class="site-nav">
+        <div class="trigger">
+            <%= link_to 'Create Post', nil, {:class => 'page-link'} %>
+      </div>
+      </nav></div>
```

Comment:
Indentation is off here. Best practice for readability is to have the opening tag and closing tag on the same indentation level.

ie
```html
<div class="wrapper">
  <p class="site-title" >Haxxor News</p>
  <nav class="site-nav">
    <div class="trigger">
      <%= link_to 'Create Post', nil, {:class => 'page-link'} %>
    </div>
  </nav>
</div>
```

Also, heads up there are two hash styles:
```ruby
# the old style
{:class => 'page-link'}
# and the newer style (introduced 11 years ago with Ruby 1.9)
{class: 'page-link'}
```

Generally it's better to use the newer style unless you have to interpolate the key.